### ISSUE TITLE
Comment position inconistency between class method and function

### ISSUE THREAD

**Issue by Alphapage** [prettier/prettier#2816]
**Prettier Version:** 1.6.1 / master

**Input:**

([playground](https://prettier.io/playground/#%7B%22content%22%3A%22function%20test(arg1%2C%2F%2F%20comment%5Cn%20%20%20%20%20%20%20%20%20%20%20%20%20%20arg2)%20%7B%7D%5Cn%5Cnclass%20cl%7B%5Cn%20%20test(arg1%2C%2F%2F%20comment%5Cn%20%20%20%20%20%20%20%20%20%20%20%20%20%20arg2)%20%7B%7D%5Cn%7D%22%2C%22options%22%3A%7B%22printWidth%22%3A80%2C%22tabWidth%22%3A2%2C%22singleQuote%22%3Afalse%2C%22trailingComma%22%3A%22none%22%2C%22bracketSpacing%22%3Atrue%2C%22jsxBracketSameLine%22%3Afalse%2C%22parser%22%3A%22babylon%22%2C%22semi%22%3Atrue%2C%22useTabs%22%3Afalse%2C%22doc%22%3Afalse%2C%22ast%22%3Afalse%7D%7D))

```js
function test(arg1,// comment
              arg2) {}

class cl{
  test(arg1,// comment
              arg2) {}
}
```

**Output:**
```js
function test(
  arg1, // comment
  arg2
) {}

class cl {
  test(arg1, arg2) {} // comment
}
```
**Expected:**
```js
function test(
  arg1, // comment
  arg2
) {}

class cl {
  test(
    arg1, // comment
    arg2
  ) {}
}
```

Notice how the comment stays inside the parameters for the function, but is moved outside for the class method, which is a bit inconsistent.

**Comment by lydell** [comment]
Everything looks as expected in the Playground. It looks like something has happened to your "Expected behavior" code snippet though – that's not how you'd like it to be formatted, is it? Can you please look through your report and clarify what your problem is? Thanks.

**Comment by Alphapage** [comment]
Sorry, it is not very explicit: the comment is at the wrong in class method.

**Comment by azz** [comment]
I added a new expected block.

**Comment by Alphapage** [comment]
It seems the wrong behaviour for comments is changing when the function body is empty or not:
https://prettier.io/playground/#%7B%22content%22%3A%22function%20t()%2F%2Ftest%5Cn%7Breturn%7D%22%2C%22options%22%3A%7B%22ast%22%3Afalse%2C%22bracketSpacing%22%3Atrue%2C%22doc%22%3Afalse%2C%22jsxBracketSameLine%22%3Afalse%2C%22output2%22%3Afalse%2C%22parser%22%3A%22babylon%22%2C%22printWidth%22%3A80%2C%22semi%22%3Atrue%2C%22singleQuote%22%3Afalse%2C%22tabWidth%22%3A2%2C%22trailingComma%22%3A%22none%22%2C%22useTabs%22%3Afalse%7D%7D

**Comment by duailibe** [comment]
This was fixed by #3441: [playground link](https://deploy-preview-3441--prettier.netlify.com/playground/#N4Igxg9gdgLgprEAuEAzArlMMCW0AE8AzjABQCGATgOYCMANAPSP6QC2bCMAOlPvwMGCq1AEwBKfMHIBfXrzAAbckSKtFwXv2JkRDZqwgcuWoWf4iJU2bzlReIeiAgAHXNCLJQVShADuAApUCJ4o5ABuEDgAJo4gKjDIqOSKRHBOAEaU5GAA1nAwAMouOThQ1MgwlOjpINEQYEkpaU5laZQwAdnUbORNqbUAVkQAHgBC2XkFheScADJlcP0tzugwLmuiy7UllO3IIBnkGQCeitBxLpRlMADqMTAAFsgAHAAMTlcQabfZLgdXODtcJLJyUOAAR3QOHBXXIPT6SGSAycaTYOEq1VqRDK1EUcAAiugIPBtk4YMd7tEnshROTsjhFLiAMJGXoHKDQUEgdBpAAqx1CyLSMhkQA)
